webapp/components/elements/FormInput.tsx

```diff
@@ -64,26 +64,34 @@ const ReceiptInput: React.FC<Props> = ({
       };
 
       return multiLine ? (
-        <Textarea
-          onBlur={(event) =>
-            props.input.onBlur(
-              event as React.FocusEvent<HTMLInputElement, Element>
-            )
-          }
-          minRows={2}
-          {...sharedProps}
-        />
+        <div>
+          <Textarea
+            id={name}
+            {...sharedProps}
+            placeholder=" "
+            onBlur={(event) =>
+              props.input.onBlur(
+                event as React.FocusEvent<HTMLInputElement, Element>
+              )
+            }
+            minRows={2}
+          />
+          <label htmlFor={name}>
+            {label} {required && '*'}
+          </label>
+        </div>
       ) : (
-        <>
+        <div>
           <Input
+            id={name}
+            {...sharedProps}
+            placeholder=" "
             onBlur={(event) =>
               props.input.onBlur(
                 event as React.FocusEvent<HTMLInputElement, Element>
               )
             }
             type={type}
-            list={name + 'list'}
-            {...sharedProps}
           />
           {suggestions && (
             <datalist id={name + 'list'}>
```

Comment:
When you're removing the list={} attribute you could also remove the id={} attribute as they were referencing each other
```suggestion
            <datalist>
```